contracts/VestingUpgradeable_V2.sol

```diff
@@ -0,0 +1,194 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.10;
+
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import "@openzeppelin/contracts/utils/math/Math.sol";
+import "@openzeppelin/contracts-upgradeable/token/ERC20/utils/SafeERC20Upgradeable.sol";
+import "./interfaces/IVesting_V2.sol";
+import "./MyToken.sol";
+import "hardhat/console.sol";
+
+/// @title VestingUpgradeable_V2
+/// @author Applicature
+/// @notice This Smart Contract provides the vesting of tokens that will be unlocking until a certain date
+/// @dev This Smart Contract provides the vesting of tokens that will be unlocking until a certain date
+contract VestingUpgradeable_V2 is IVesting_V2, OwnableUpgradeable {
+    using SafeERC20Upgradeable for MyToken;
+    using Math for uint256;
+
+    /// @notice Store constant of percentage 100
+    /// @dev Store constant of percentage 100
+    uint256 public constant MAX_INITIAL_PERCENTAGE = 100;
+
+    /// @notice Store the initial timestamp in unix
+    /// @dev Store the initial timestamp in unix
+    uint256 public initialTimestamp;
+
+    /// @notice Store the last index of mapping of investors
+    /// @dev Store the last index of mapping of investors
+    uint256 public lastIndexInvestors;
```

Comment:
you are break storage slots ,https://docs.openzeppelin.com/upgrades-plugins/1.x/proxies . new variable must added to end of storage for new version not in mid